Cannot find module `better-auth/node` or its corresponding type declarations

**Describe the bug**
The `toNodeHandler` does not work. 
I get this error when i try to test it out in express
```
Cannot find module 'better-auth/node' or its corresponding type declarations
```

I tried importing from `better-auth/dist/node` and got another error

```
Package subpath './dist/node' is not defined by "exports" in /Users/baker/Desktop/node-better-auth-test/node_modules/better-auth/package.json
```

**To Reproduce**
Steps to reproduce the behavior:

1. Generate a simple express app
2. Follow the installation steps in the docs
3. Follow the node integration setup from docs
4. Run the express app
5. See error

**Expected behavior**
For the `toNodeHandler` to exist and the path `better-auth/node` to export it properly.

**Screenshots**
<img width="1083" alt="image" src="https://github.com/user-attachments/assets/f93fe07d-75f5-41ef-8c20-f2cbed65dc73">


**Desktop (please complete the following information):**

- OS: MacOs
- Browser: N/A (didn't get to try it out)
- Version [e.g. 22]

**Smartphone (please complete the following information):**
N/A

**Additional context**
N/A

--- a/dev/bun/tsconfig.json
+++ b/dev/bun/tsconfig.json
@@ -9,7 +9,7 @@
 		"allowJs": true,
 
 		// Bundler mode
-		"moduleResolution": "bundler",
+		"moduleResolution": "Node",
 		"allowImportingTsExtensions": true,
 		"verbatimModuleSyntax": true,
 		"noEmit": true,

--- a/packages/better-auth/package.json
+++ b/packages/better-auth/package.json
@@ -138,6 +138,79 @@
       "require": "./dist/adapters/mongodb.cjs"
     }
   },
+  "typesVersions": {
+    "*": {
+      "*": [
+        "./dist/index.d.ts"
+      ],
+      "node": [
+        "./dist/node.d.ts"
+      ],
+      "react": [
+        "./dist/react.d.ts"
+      ],
+      "vue": [
+        "./dist/vue.d.ts"
+      ],
+      "svelte": [
+        "./dist/svelte.d.ts"
+      ],
+      "social-providers": [
+        "./dist/social.d.ts"
+      ],
+      "client": [
+        "./dist/client.d.ts"
+      ],
+      "client/plugins": [
+        "./dist/client/plugins.d.ts"
+      ],
+      "types": [
+        "./dist/types.d.ts"
+      ],
+      "crypto": [
+        "./dist/crypto.d.ts"
+      ],
+      "cookies": [
+        "./dist/cookies.d.ts"
+      ],
+      "oauth2": [
+        "./dist/oauth2.d.ts"
+      ],
+      "solid": [
+        "./dist/solid.d.ts"
+      ],
+      "api": [
+        "./dist/api.d.ts"
+      ],
+      "db": [
+        "./dist/db.d.ts"
+      ],
+      "plugins": [
+        "./dist/plugins.d.ts"
+      ],
+      "plugins/access": [
+        "./dist/access.d.ts"
+      ],
+      "svelte-kit": [
+        "./dist/svelte-kit.d.ts"
+      ],
+      "solid-start": [
+        "./dist/solid-start.d.ts"
+      ],
+      "next-js": [
+        "./dist/next-js.d.ts"
+      ],
+      "adapters/prisma": [
+        "./dist/adapters/prisma.d.ts"
+      ],
+      "adapters/drizzle": [
+        "./dist/adapters/drizzle.d.ts"
+      ],
+      "adapters/mongodb": [
+        "./dist/adapters/mongodb.d.ts"
+      ]
+    }
+  },
   "devDependencies": {
     "@prisma/client": "^5.19.1",
     "@simplewebauthn/types": "^10.0.0",

